android/app/build.gradle

```diff
@@ -103,18 +103,25 @@ def getVersionName = { ->
 
 android {
     compileSdkVersion 24
-    buildToolsVersion "23.0.1"
+    buildToolsVersion "24.0.2"
 
     defaultConfig {
         applicationId "im.status.ethereum"
         minSdkVersion 18
-        targetSdkVersion 23
+        targetSdkVersion 24
```

Comment:
Are we OK bumping this? There was a discussion a few weeks ago when I bumped this to 25 that it's better if we keep it as low as possible for compatibility. Don't know how devices/environments would be impacted by this though.

Other than that, LGTM. Nice one!